classes/classes/Items/Mutations.as

```diff
@@ -1582,6 +1582,416 @@
 			player.refillHunger(20);
 		}
 
+// Fuck yo dog shit we full-on wolf bitches now -Foxwells
+public
+function wolfPepper(type: Number, player: Player): void {
+    var tfSource: String = "wolfPepper";
+    var temp2: Number = 0;
+    var temp3: Number = 0;
+    var crit: Number = 1;
+    changes = 0;
+    changeLimit = 1;
+    if (type == 1) changeLimit += 2;
+    if (rand(2) == 0) changeLimit++;
+    if (rand(2) == 0) changeLimit++;
+    if (player.findPerk(PerkLib.HistoryAlchemist) >= 0) changeLimit++;
+    if (player.findPerk(PerkLib.TransformationResistance) >= 0) changeLimit--;
+    outputText("The pepper has an uncomfortable texture to it, being covered in soft fuzz like it's a peach but somewhat crunchy like any other pepper. Its spiciness makes you nearly spit it out, and you're left sniffling after.", false);
+    if (rand(100) < 15) {
+        crit = int(Math.random() * 20) / 10 + 2;
+        outputText(" Maybe it was a bit too spicy... The pepper seemed far more ripe than what you'd expect.", false);
+    }
+    //STAT CHANGES - TOU SPE INT RANDOM CHANCE, LIB LUST COR ALWAYS UPPED
+    dynStats("lib", 1 + rand(2), "lus", 5 + rand(10), "cor", 1 + rand(5));
+    outputText("\n\nYou lick your lips after you finish. That spiciness hit you in more ways than one.", false);
+    if (player.tou < 70 && rand(3) == 0 && changes < changeLimit) {
+        dynStats("tou", (1 * crit));
+        if (crit > 1) outputText("\n\nYou roll your shoulders and tense your arms experimentally. You feel more durable, and your blood seems to run through you more clearly. You know you have more endurance.", false);
+        else outputText("\n\nYour muscles feel denser and more durable. Not so much that feel stronger, but you feel like you can take more hits.", false);
+        changes++;
+    }
+    if (player.spe > 30 && rand(3) == 0 && changes < changeLimit) {
```

Comment:
I was studying wolf hunting patterns. They are not very speedy, but they are intelligent and have lots of endurance (hence to toughness raise). It wasn't that they're clumsy, they just don't run fast. I will reconsider the stat changes though.